### PR TITLE
RxSwift burndown 🔥

### PR DESCRIPTION
**Pull request by Abdul-Moiz**
Whats new:
- Removed RxSwift from playground. 
- Converted to either classic add target actions or Combine.
- Some of the changes related to `self` addition/removal is from `swiftformat`

Tests:
- Manually tested playgrounds and fixed few bugs/crashes.

### CODE REVIEW

File: Thumbprint/Targets/Thumbprint/Components/CalendarPicker/DayOfWeekHeaderView.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ class DayOfWeekHeaderView: UIView {
             return label
         })
 
-        stackView = UIStackView(arrangedSubviews: labels)
+        self.stackView = UIStackView(arrangedSubviews: labels)
```
Comment (millermjared) [inline Thumbprint/Targets/Thumbprint/Components/CalendarPicker/DayOfWeekHeaderView.swift:21]: This change made me curious - what's up? I don't see a local declaration or anything - why disambiguate with self here?

**Review by millermjared — DISMISSED** [review]
Nice cleanup work! I had one very minor curiosity question, lgtm!

File: Thumbprint/Targets/Playground/Classes/Inspectable/InspectableViews/Label+InspectableView.swift [hunk 2]
```diff
@@ -57,6 +56,7 @@ extension Label: InspectableView {
         label.text = "This is a label"
         label.setContentCompressionResistancePriority(.required, for: .horizontal)
         label.setContentHuggingPriority(.required, for: .horizontal)
+        label.isUserInteractionEnabled = true
```
Comment (scottasoutherland) [inline Thumbprint/Targets/Playground/Classes/Inspectable/InspectableViews/Label+InspectableView.swift:59]: Why was this change necessary ?

Comment (Abdul-Moiz) [reply Thumbprint/Targets/Playground/Classes/Inspectable/InspectableViews/Label+InspectableView.swift:59]: As label are by default not interactive. This was preventing it from dragging/dropping in the view.

File: Thumbprint/Targets/Playground/Classes/Inspectable/InspectableProperties/ButtonActionInspectableProperty.swift [hunk 3]
```diff
@@ -10,15 +8,14 @@ class ButtionActionInspectableProperty: InspectableProperty {
     }
 
     private var button: Button
-    private let disposeBag = DisposeBag()
 
     init(buttonTitle: String, buttonAction: @escaping () -> Void) {
         let buttonSize = Button.Size(textStyle: .title6, contentPadding: .init(width: 12.0, height: 8.0))
-        button = Button(theme: .primary, size: buttonSize)
+        self.button = Button(theme: .primary, size: buttonSize)
```
[the remark this answers is not in the document]
Comment (Abdul-Moiz) [reply Thumbprint/Targets/Playground/Classes/Inspectable/InspectableProperties/ButtonActionInspectableProperty.swift:14]: There are lot more changes with `swiftformat` on iOS repo. Lots of them related to self. Not sure why that is so important to add every where.

**Review by scottasoutherland — APPROVED** [review]
